client/protocols/unlock.py

```diff
@@ -172,6 +203,137 @@ def interact(self, unique_id, case_id, question_prompt, answer, choices=None, ra
     # Private Methods #
     ###################
 
+    # Displaying the correct messages depending on students' wrong answers and treatment group ID
+    def error_handling(self, unique_id, input_lines,access_token):
+
+        shorten_unique_id = unique_id[unique_id.index('\n')+1:]
+
+        # Try to get th info dictionary for this question
+        wa_2_dict_info = json_dict_file['dictAssessId2WA2DictInfo'].get(shorten_unique_id)
+
+        if not wa_2_dict_info: # get returns None if wa_2_dict_info doesn't have shorten_unique_id in dictionary
+            print ("-- Not quite. Try again! --")
+            return ({}, -1)
+
+        dict_info = wa_2_dict_info.get(repr(input_lines))
+
+        # If this wrong answer is not in the JSON file, display default message
+        if not dict_info:            
+            print ("-- Not quite. Try again! --")
+            return ({}, -1)
+
+        # Checks to see the student currently has a treatment group number. If not, calls helper function in auth.py
+        if not os.path.isfile("tests/tg.ok_tg"):
```

Comment:
After discussion with @Sumukh , this can be a constant at the top of the guidance protocol.